### ISSUE TITLE
Focus styles: Review and finalize designs

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/vets-design-system-documentation#1315]
## Context

The VA.gov Design System needs updated focus styles for better accessibility because the current styles don't meet color contrast guidelines in the Web Content Accessibility Guidelines. In updating the styles, we want to also think ahead to future versions of WCAG, which likely have more stringent requirements for focus styles.
Specifically, the current focus style fails to meet compliance in the scenario of white button on dark background (e.g. the active search button).

Links:
- [Success Criterion 1.4.11 Non-text Contrast](https://www.w3.org/TR/WCAG21/#non-text-contrast)
- [What's New in WCAG 2.2?](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/)
- [2.4.11 Focus Appearance(AA)](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#2411-focus-appearance-aa)
- [2.4.12 Focus Not Obscured (Minimum) (AA)](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#2412-focus-not-obscured-minimum-aa)
- [Testing, Evaluation, and Documentation: Double box-shadow focus styling](https://docs.google.com/document/d/1o-bfmdz6arPju9XCiGn-NR9UpgoN1Vp0EB3cvRGMA3Q/edit?usp=sharing)
- [VA Design System Focus Styles: Questions and Assumptions](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/shared-support/accessibility/projects/design-system-focus-styles/focus-styles-questions-assumptions-requirements.md)
- [Use double box-shadow for focus styling #654](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/654)

## Work to be done

This work is to present and confirm the final color and alternative focus methods (if needed) for focus styles with stakeholders on the design system team and within the VA.gov ecosystem.

## Acceptance criteria

- [ ] Artifacts have been presented to the design system team.
- [ ] The color and alternative focus method (if necessary) is selected.

**Comment by caw310** [comment]
@davidakennedy @rsmithadhoc is this issue still needed? If so, should we be more specific about what work is needed?

**Comment by davidakennedy** [comment]
@caw310 I would keep it. Once #1314 is done, I can move to this one to get approval.

**Comment by davidakennedy** [comment]
We ended up not needing this ticket. The bulk of the design work happened in the other one. I've [left an update there and closed it out](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1314#issuecomment-2613375808). This one can be closed too. If the design system team needs to do more work, it can open new tickers. cc: @caw310
